src/main/java/ch/heigvd/Client.java

```diff
@@ -52,9 +52,10 @@ public void run() {
                     continue;
                 }
 
-                if (word.isEmpty() && arguments[0].equalsIgnoreCase(START)) {
+                if (arguments[0].equalsIgnoreCase(START)) { // TODO : pk IgnoreCase ? (toUpperCase ligne 47)
```

Comment:
probably a superfluous check that could be replaced by ```arguments[0].equals(START)``` I wasn't sure how we wanted to handle characters such as éèà etc